vnext/Microsoft.ReactNative/DevMenuControl.xaml

```diff
@@ -125,6 +125,21 @@
                 <TextBlock Grid.Column='1' Grid.Row='1' FontSize='12' Opacity='0.5' TextWrapping='Wrap'>Will bring up an overlay that lets you tap on any UI element and see information about it</TextBlock>
             </Grid>
         </Button>
+        <Button HorizontalAlignment='Stretch' HorizontalContentAlignment='Stretch' x:Name='SamplingProfiler' Style='{StaticResource ButtonRevealStyle}'>
+          <Grid HorizontalAlignment='Stretch'>
+            <Grid.ColumnDefinitions>
+              <ColumnDefinition Width='Auto'/>
+              <ColumnDefinition Width='*'/>
+            </Grid.ColumnDefinitions>
+            <Grid.RowDefinitions>
+              <RowDefinition/>
+              <RowDefinition/>
+            </Grid.RowDefinitions>
+            <FontIcon Grid.Column='0' Grid.Row='0' Grid.RowSpan='2' VerticalAlignment='Top' FontFamily='{StaticResource SymbolThemeFontFamily}' Foreground='{StaticResource SystemControlForegroundAccentBrush}' Margin='8,8,16,8' Glyph='&#xE773;'/>
```

Comment:
consider using a better/unique icon, like the Record icon (`&#xE7C8`). You can switch it to the stop icon once it's playing :) (`&#xE71A`)